### PR TITLE
Add option to refresh the index.json

### PR DESCRIPTION
**Pull request by ArtskydJ**

### CODE REVIEW

File: test/index_manager_persistence.js [hunk 1]
```diff
@@ -52,3 +52,38 @@ test("Loading the index from levelUP instead of the retrieval object", function(
...
+
+		indexManager.getPosts(function(err, posts) { // Get the posts again, expect cached posts
+			t.ifError(err)
+			t.equal(posts.length, 1)
+
+			indexManager.refresh(function (err, posts) { // Refresh the posts (testing this functionality)
+				t.ifError(err)
+				t.equal(posts.length, 2)
+
+				indexManager.refresh(function (err, posts) { // Get the posts yet again, expect them to be refreshed
```
Comment (TehShrike) [inline test/index_manager_persistence.js]: Was this supposed to be a `getPosts` call?

Comment (ArtskydJ) [inline test/index_manager_persistence.js]: yup

**Comment by TehShrike** [comment]
w00t!

**Comment by TehShrike** [comment]
Published as 2.5.0

**Pull request merged**